src/py/flwr/client/app.py

```diff
@@ -612,6 +612,12 @@ def _on_backoff(retry_state: RetryState) -> None:
                     send(reply_message)
                     log(INFO, "Sent reply")
 
+                except RunNotRunningException:
+                    log(INFO, "")
+                    log(INFO, "Run ID %s stopped.", run_id)
```

Comment:
The `Run` doesn't have to be stopped, it could be also finished for another reason.

Reply:
Thanks, you're right. I've rephrased the info message.